crates/libduckdb-sys/upgrade.sh

```diff
@@ -1,33 +1,45 @@
-#!/bin/bash -e
+#!/bin/bash
+
+set -e
+
+SCRIPT=$(realpath "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT")
 
-SCRIPT_DIR=$(cd "$(dirname "$_")" && pwd)
-CUR_DIR=$(pwd -P)
 echo "$SCRIPT_DIR"
-cd "$SCRIPT_DIR" || { echo "fatal error" >&2; exit 1; }
+cd "$SCRIPT_DIR"
 cargo clean
 mkdir -p "$SCRIPT_DIR/../../target" "$SCRIPT_DIR/duckdb"
 export DUCKDB_LIB_DIR="$SCRIPT_DIR/duckdb"
-export DU_INCLUDE_DIR="$DUCKDB_LIB_DIR"
 
 # Download and extract amalgamation
-DUCKDB_VERSION=v1.0.0
+# todo: fix this version
+DUCKDB_VERSION=v1.1.0
 git submodule update --init --checkout
-cd "$SCRIPT_DIR/duckdb-sources" || { echo "fatal error" >&2; exit 1; }
+cd "$SCRIPT_DIR/duckdb-sources"
 git fetch
 git checkout "$DUCKDB_VERSION"
-cd "$SCRIPT_DIR" || { echo "fatal error" >&2; exit 1; }
+cd "$SCRIPT_DIR"
 python3 "$SCRIPT_DIR/update_sources.py"
 
+# Regenerate bindgen file for DUCKDB
+cd "$SCRIPT_DIR"
+rm -f "$SCRIPT_DIR/src/bindgen_bundled_version_loadable.rs"
+find "$SCRIPT_DIR/../../target" -type f -name bindgen.rs -exec rm {} \;
+cargo build --features "extensions-full buildtime_bindgen loadable_extension"
+find "$SCRIPT_DIR/../../target" -type f -name bindgen.rs -exec cp {} "$SCRIPT_DIR/src/bindgen_bundled_version_loadable.rs" \;
+
+# Sanity checks
+# FIXME: how to test this here?
```

Comment:
🙈

Reply:
iirc some of the tests are bindgen struct layout validations, not sure if these were failing or not

Reply:
Ok I will take a look, I think the problem here was that when building with the loadable_extension variant of the bindings, the tests have no way to create a duckdb instance making it basically impossible to run most of the tests